src/lang/pt-BR/TreeView.json

```diff
@@ -0,0 +1,4 @@
+{
+  "tableHeader": "Vista de árvore",
+  "tableSubHeader": "Dados personalizáveis ​​baseados em colunas para todas as tarefas desta suíte"
```

Comment:
Grammatically correct, but sounds like "The view from a tree". I think for `tableHeader` "Visualização em Árvore". And for the `tableSubHeader` maybe "Dados em formato colunar e customizáveis disponíveis para as tarefas desta suite." ("suíte" is the correct translation, but we still use "suite" as in "suite de testes" / "test suite" and "suíte" more for a hotel room name... can't explain why :confused: )

Reply:
"The view from a tree" - he heh, I like that.  I love being up a tree, looking out at the landscape. :grin: